src/pyedb/dotnet/edb_core/utilities/simulation_setup.py

```diff
@@ -100,7 +100,11 @@ def __init__(self, pedb, edb_object=None):
 
     @property
     def sim_setup_info(self):
-        return SimSetupInfo(self._pedb, sim_setup=self, edb_object=self._edb_object.GetSimSetupInfo())
+        if self.type not in ["hfss_pi", "raptor_x"]:
+            return SimSetupInfo(self._pedb, sim_setup=self, edb_object=self._edb_object.GetSimSetupInfo())
```

Comment:
why do you check this property? This line is exactly the same as the original code.

Reply:
Because RaptorX and most likely HFSSPI don't have GetSimSetupInfo method in EDB